packages/react/src/components/Pagination/Pagination.tsx

```diff
@@ -91,6 +91,7 @@ export const Pagination = forwardRef<HTMLElement, PaginationProps>(
     return (
       <nav
         ref={ref}
+        aria-label='Pagination'
```

Comment:
```suggestion
        aria-label='Paginering'
```

Reply:
We had a discussion about this yesterday, and found that other websites use english for this. So to not confuse users used to hearing "pagination", we went with english. 
uutilsynet also does it in english: [https://www.uutilsynet.no/tilsynsrapportar/rapportar-fra-tilsyn/1167?f%5B0%5D=gt%3A9](https://www.uutilsynet.no/tilsynsrapportar/rapportar-fra-tilsyn/1167?f%5B0%5D=gt%3A9)

Reply:
Alrighty :)